src/filename_inspector.py

```diff
@@ -69,28 +76,32 @@ def run():
         violations = find_non_matching_files(name_patterns, paths, excludes)
         violation_count = len(violations)
 
-        set_output('violation_count', str(violation_count))
+        set_action_output('violation-count', str(violation_count))
+
+        if verbose_logging:
+            print(f'Total violations: {violation_count}')
```

Comment:
What's the difference of `print` and `logging`?

Reply:
Print output is independent of logging settings.
What level would you propose?

Reply:
That's extra weird. There are good chances, that verbose set to `true` will print out the same info twice.
My suggestion:
* treat `console` as _console only_ output, but print there even if other option is used (and adjust descripton accordingly)
* if `verbose` is _false_, print only the violations number, if _true_ print all violations

Or use other logic. But this is IMHO wrong.

Reply:
Improved.